tera/src/snapshot_tests/rendering_inputs/success/spread.txt

```diff
@@ -0,0 +1,9 @@
+{% set base = {"a": 1, "b": 2} %}
+{% set other = {"c": 3} %}
+Simple spread: {{ {...base} | safe }}
+Add key: {{ {...base, "d": 4} | safe }}
+Multiple spreads: {{ {...base, ...other} | safe }}
+Override: {{ {...base, "a": 99} | safe }}
```

Comment:
Just now looking at this. What happens in this scenario?

```jinja
{% set base1 = {"a": 1, "b": 2} %}
{% set base2 = {"a": 3, "b": 4} %}
Override 2: {{ {...base1,...base2} | safe }}
```

I think in JS/TS `base2` would override any shared fields it has with `base1`. Might be useful to add as a test case even as documentation.